IOStream/FileInputStream.m

```diff
@@ -36,8 +37,24 @@ - (void)dealloc {
     [self close];
 }
 
+- (void)enableCopyBuffer:(BOOL)enable {
```

Comment:
I threw this on FileInputStream in a rush trying to get things to work and then forgot about it.  I can change it as you have suggested.  The only thing I think I would need to add to FileInputStream is the ability to read what position the seek pointer is at before I rewind the file ( a getpos() method) because the header is not a fixed length.